release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,44 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-improved-operator-installation-workflow"]
+==== Improved Operator installation workflow with OperatorHub
+
+When administrators install Operators with OperatorHub, they now get immediate
+feedback to ensure that the Operator is installing properly.
+
+[id="ocp-4-6-web-console-improved-operand-details-view"]
+==== Improved Operand details view
+
+You can now see the schema grouping of `specDescriptor` fields and the status of
+your Operands on the Operand's details view, so that you can easily see the
+status and configure the `spec` of the Operand instance.
+
+[id="ocp-4-6-web-console-view-operators-related-objects"]
+==== View cluster Operators' related objects
```

Comment:
If we have this in the release notes, I'd also mention https://issues.redhat.com/browse/CONSOLE-2202, which adds a managed by link for operands owned by OLM operators.